include/bluetooth/hci.h

```diff
@@ -2053,6 +2071,23 @@ struct bt_hci_evt_le_req_peer_sca_complete {
 	uint8_t  sca;
 } __packed;
 
+#define BT_HCI_EVT_LE_BIGINFO_ADV_REPORT        0x22
+struct bt_hci_evt_le_biginfo_advertising_report {
```

Comment:
Maybe consider shortening "advertising" to "adv"

Reply:
Agreed. Done